src/widget/transform.ts

```diff
@@ -1,21 +1,29 @@
 import type { Widget } from "../types";
 
-export const getName = (name: string, capitalize: boolean) =>
-  capitalize ? toCapitalized(name) : name;
-
-export const getWidgetName = (widget: Widget, useLabel: boolean, capitalize: boolean): string =>
+export const getName = (name: string, capitalize: boolean, delimiter: string) =>
+  toDelimiter(capitalize ? toCapitalized(name) : name, delimiter);
```

Comment:
@jsaari97 I've changed approach a bit here and made sure to replace any existing delimiters too. With this you really get uniform type names. E.g. `kitchenSink_typed_list_type_2_object` with default options and where `type_2_object` is an actual name, can now with...
```javascript 
{ label:true, capitalize: true, delimiter: "" }
```
...become `KitchenSinkTypedListType2Object`.

WDYS?